src/vma/sock/sockinfo_tcp.cpp

```diff
@@ -2011,6 +2011,18 @@ int sockinfo_tcp::bind(const sockaddr *__addr, socklen_t __addrlen)
 		return -1; //error
 	}
 
+	if (!m_bound.is_anyaddr() && !m_bound.is_local_loopback()) {
```

Comment:
@rosenbaumalex 
we should provide limitation note for vma_poll() usage as `SO_BINDTODEVICE usage is limited if application wants to get 'ring' before connect()/listen()`

`m_so_bindtodevice_ip != 0` can be useful for `setsockopt(SO_BINDTODEVICE)->bind()->get_socket_rings_fds()` flow but can not help for other as `bind()->setsockopt(SO_BINDTODEVICE)->get_socket_rings_fds()`